components/src/Button/Button.js

```diff
@@ -92,7 +92,13 @@ const size = ({ size = 'medium' }) => (({
   `
 })[size]);
 
-const Button = styled.button`
+const Wrapper = styled.span`
```

Comment:
@nikola-nulogy I added this wrapper component to children when there are more than one (ie I don't add it to a single icon on its own or a button with only text).

The alignment and size of the button with an icon seems wrong though.

If you runtime storybook locally, then check out these stories:

[📖 With an icon and text](http://localhost:8080/?selectedKind=Buttons%20%2F%20Button&selectedStory=With%20an%20icon%20and%20text&full=0&addons=0&stories=1&panelRight=1&addonPanel=PeterPanen%2Fstorybook-addon-scissors%2Fpanel)

the alignment seems okay, but my inspector is showing this button as `33px` (I didn't check the dimensions for the other button sizes).

<img width="317" alt="screen shot 2018-09-18 at 1 09 30 pm" src="https://user-images.githubusercontent.com/2694/45704126-2358ea00-bb44-11e8-80dd-b33719a893a5.png">

[📖 With an icon](http://localhost:8080/?selectedKind=Buttons%20%2F%20Button&selectedStory=With%20an%20icon&full=0&addons=0&stories=1&panelRight=1&addonPanel=PeterPanen%2Fstorybook-addon-scissors%2Fpanel)

The size is off again: I read `30px`, and the icon is not aligned.

<img width="312" alt="screen shot 2018-09-18 at 1 10 33 pm" src="https://user-images.githubusercontent.com/2694/45704225-57cca600-bb44-11e8-87d4-c8e3fee9b8f5.png">